src/scripts/main.js

```diff
@@ -1,3 +1,36 @@
 'use strict';
 
-// write your code here
+const promise1 = new Promise((resolve) => {
+  const logo = document.querySelector('.logo');
+
+  logo.addEventListener('click', () => {
+    resolve();
+  });
+});
+
+// eslint-disable-next-line promise/param-names
+const promise2 = new Promise((_, reject) => {
+  setTimeout(() => {
+    // eslint-disable-next-line prefer-promise-reject-errors
+    reject();
```

Comment:
The `reject` function is called without any error message or object. It's a good practice to provide an error message or object to help with debugging and understanding the rejection reason.